src/Robin/Ntlm/Crypt/Random/OpenSslRandomByteGenerator.php

```diff
@@ -15,7 +15,7 @@
  * "openssl" extension.
  *
  * @link http://php.net/openssl
- * @deprectated NOTE! This implementation is deprecated, as it's been found to
+ * @deprecated Insecure! This implementation is deprecated, as it's been found to
```

Comment:
It's not a tag, though. It's a message for the deprecation. If we wanted to move this to the docblock body, `NOTE:` would make more sense I think.